ts/src/index.ts

```diff
@@ -128,9 +128,10 @@ async function initConfigMetadata(config: ProfilerConfig):
  */
 export async function createProfiler(config: Config): Promise<Profiler> {
   if (!semver.satisfies(process.version, pjson.engines.node)) {
-    throw new Error(
-        `Could not start profiler: node version ${process.version}` +
-        ` does not satisfies "${pjson.engines.node}"`);
+    logWarning(
```

Comment:
Can we make it a warning (or just skip the check) for versions with pre-release tag, but keep it an error for released versions?